src/Apps/CollectorProfile/Routes/Saves2/CollectorProfileSaves2Route.tsx

```diff
@@ -86,7 +86,11 @@ export const CollectorProfileSaves2RouteFragmentContainer = createFragmentContai
           first: 30
           default: false
           saves: true
-        ) {
+        )
+          @connection(
+            key: "CollectorProfileSaves2Route_otherSaves"
+            filters: []
```

Comment:
Is the empty `filters` argument needed?

Reply:
Good question, I should have left a comment (maybe its worth adding the comment to the codebase itself).

There was a gotcha here that @dimatretyak helped me with, I had the updater implementation without including this here and the result was that the ui was never updating. 

You can test it out if you comment that out and try to create a new list you'll see that the UI is not being updated. Not sure why but needs to be there.

@dimatretyak correct me if I am wrong, and do you think its worth adding a comment here?

Reply:
On the delete flow you will see the update on the ui but I think it is because of the routing redirect that we do

Reply:
@anandaroop by default, all arguments and their values (in our case it is `first: 30`, `default: false`, `saves: true`) are used to generate the special "key" by which we can then find/identity the connections we need among others. 

And based on the 👆 information we have 2 approaches:
* specify `filters: []` and search for the desired connection **only** by the key (as George did it [here](https://github.com/artsy/force/blob/02d91ef25623a00cc69feff84b5984abb3b13391/src/Apps/CollectorProfile/Routes/Saves2/Components/Actions/Mutations/useCreateCollection.ts#L21))
* OR ignore `filters` and pass all arguments and their values to the `getConnection` (see example below). IMO This method is more complicated

```javascript
const otherSavesConnection = ConnectionHandler.getConnection(me, key, {
  first: 30,
  default: false,
  saves: true,
})
```

You can read more about it here: https://relay.dev/docs/guided-tour/list-data/updating-connections/#connection-identity-with-filters

Reply:
Thanks for the explanation